src/database/datatools/nomass_materials.py

```
@@ -0,0 +1,142 @@
+    des_data = [material_id] + dt[:-1] 
+    update_description_nomass_material(db_path, des_data)
+    if name:
+        update_description_all_materials(db_path, [am_id, name, 'NoMass', None, material_id])
+
+def delete_nomass_material(db_path: str, nomass_id: int) -> None:
+    conn = sqlite3.connect(db_path)
+    cursor = conn.cursor()
+
+    table_name = "no_mass_materials"
+
+    sql = f"DELETE FROM {table_name} WHERE id = ?"
```

Ruff (0.14.14)

[error] Possible SQL injection vector through string-based query construction

(S608)